src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/DecrypterImpl.java

```diff
@@ -96,7 +96,7 @@ public BlobApplicationAware decrypt(BlobApplicationAware blob) {
   }
 
   protected void decryptFile(InputStream input, OutputStream output, String blobName)
-      throws IOException, PGPException {
+      throws IOException, PGPException, NullPointerException {
```

Comment:
is this necessary to specify in the throws clause? as far as i remember you are forced to declare only the checked Exception (i.e. not  inherited from RuntimeException)

Reply:
You are right, the NullPointerException was a refuse of the previous exception handling strategy.
Fixed, thanks!